doc/userguide/rules/dns-keywords.rst

```diff
@@ -46,6 +48,47 @@ Match on DNS requests where the **opcode** is NOT 0::
 
   dns.opcode:!0;
 
+Match on DNS requests where the **opcode** is between 7 and 15, exclusively:
+
+  dns.opcode:7-15;
+
+Match on DNS requests where the **opcode** is not between 7 and 15:
+
+  dns.opcode:!7-15;
+
+dns.rcode
+----------
+
+This keyword matches on the **rcode** found in the DNS header flags.
+It uses a 8-bit unsigned integer as value.
+
+Syntax
+~~~~~~
+
+::
+
+   dns.rcode:[!]<number>
+   dns.rcode:[!]<number1>-<number2>
+
+Examples
+~~~~~~~~
+
+Match on DNS requests and responses with **rcode** 4::
+
+  dns.rcode:4;
+
+Match on DNS requests where the **rcode** is NOT 0::
+
+  dns.rcode:!0;
+
+Match on DNS requests where the **rcode** is between 7 and 15, exclusively:
```

Comment:
Are 7 and 15 included? If so, suggest "between 7 and 15, inclusively"; exclusive implies something differrent.

[7-15] -- inclusive
[7-15) -- exclusive (15 not included)